cf-agent/vercmp.c

```diff
@@ -48,13 +48,18 @@ static VersionCmpResult InvertResult(VersionCmpResult result)
 
 static VersionCmpResult AndResults(VersionCmpResult lhs, VersionCmpResult rhs)
 {
+    // These assertions were added, because the previous definition of
+    // VersionCmpResult relied on these assumptions:
+    assert(VERCMP_MATCH == true && true == 1);
+    assert(VERCMP_NO_MATCH == false && false == 0);
```

Comment:
I these asserts are not necessary. Maybe just add comments about the previous definitions next to the new ones.

Reply:
Thanks for giving me the opportunity to read the C standards :)